ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/util/BatchWrite.java

```diff
@@ -71,9 +71,9 @@ public BatchWrite(long maxBytes, int maxMessages, Duration maxDelay, String batc
 
     public Batch() {
       // wait for init then setup full indicator by timeout
-      full = init.thenRunAsync(this::timeout);
+      full = init.thenRunAsync(this::timeout).exceptionally(ignore -> null);
       // wait for full then close
-      result = full.handleAsync(this::close);
+      result = full.thenComposeAsync(this::close);
```

Comment:
`this::close` can't ignore exceptions on `full` *and* return a `CompletableFuture`, so `full` got an `exceptionally` clause to handle ignoring exceptions.